### ISSUE TITLE
button-events dependency needs version bump

### ISSUE THREAD

**Issue by bnielsen1965** [bartbutenaers/node-red-contrib-button-events#1]
The package.json button-events dependency needs the version number bumped from 1.0.0 to 1.0.1 to get the latest version with gpioChange() result messages...

```json
...
    "dependencies": {
       "button-events": "^1.0.1"
    },
...
```

**Comment by bartbutenaers** [comment]
Hi Bryan,

Did you have any problems with it?
Because I had added last week the `^` in front of the version number to make sure that all your minor versions are automatically used when my node is installed...  See [StackOverflow](https://stackoverflow.com/a/22345808).

To make sure it works correctly, I have uninstalled your and mine packages.  And when I install my package again, the latest version of your package is automatically installed:

![image](https://user-images.githubusercontent.com/14224149/95918369-f2dbec80-0dab-11eb-9f63-125b7cfe5b52.png)

Could it be that you have installed my node before you had published your latest version on NPM?  In that case you simply need to reinstall it...

Normally I create a new version of my node, to make it visible for users that something has changed (and to motivate them to install it).  But since you have published your library a few hours as me, I haven't created a new version of my node ...

Hope this makes sense?
Bart
